components/rating/index.css

```diff
@@ -10,6 +10,13 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+.spectrum--medium {
+  --spectrum-rating-icon-gap: var(--spectrum-spacing-75);
+}
+.spectrum--large {
+  --spectrum-rating-icon-gap: var(--spectrum-spacing-100);
+}
```

Comment:
Defining these here temporarily. The Web Components team prefers that we define these scale based tokens in our `/tokens` package. I can add those there tomorrow and run a release. This works for us now, though.